default.py

```diff
@@ -77,24 +80,39 @@ def _tearDown(self):
             self._tracker.join()
         self._tracker = None
 
-    def _loginMyEpisodes(self):
+    def _createMyEpisodes(self):
         username = _addon.getSetting('Username').decode('utf-8', 'replace')
         password = _addon.getSetting('Password')
 
         login_notif = _language(32912)
         if username is "" or password is "":
             notif(login_notif, time=2500)
+            self.keep_going = False
             return None
 
         mye = MyEpisodes(username, password)
-        if mye.is_logged:
-            login_notif = "%s %s" % (username, _language(32911))
-        notif(login_notif, time=2500)
-
-        if mye.is_logged and (not mye.get_show_list()):
-            notif(_language(32927), time=2500)
         return mye
 
+    def _tryLogin(self):
+        if not self.mye.is_logged:
```

Comment:
Ah, yes. That's just me being damaged from working with coding rules which only allows one return statement per function.

I couldn't find any other place where I did the same thing.